aws-record.gemspec

```diff
@@ -1,10 +1,12 @@
+# frozen_string_literal: true
+
 version = File.read(File.expand_path('../VERSION', __FILE__)).strip
 
 Gem::Specification.new do |spec|
   spec.name          = "aws-record"
   spec.version       = version
   spec.authors       = ["Amazon Web Services"]
-  spec.email         = ["alexwood@amazon.com"]
+  spec.email         = ["mamuller@amazon.com", "alexwoo@amazon.com"]
```

Comment:
Our main sdk gems don't provide an email on spec - we should probably do the same for these.